Update README.md (#11)
vtex-apps/checkout-identification: docs/README.md

> 📢 Use this project, [contribute](https://github.com/{OrganizationName}/{AppName}) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).

Reviewer: ```suggestion
📢 Use this project, [contribute](https://github.com/vtex-apps/checkout-identification) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
```

> # CHECKOUT IDENTIFICATION

Reviewer: ```suggestion
# Checkout Identification
```

> ⚠️⚠️ **This is an ongoing, unsupported, unfinished and undocumented project. We do not guarantee any results after installation, use it at your own risk.** ⚠️⚠️

Reviewer: ```suggestion
> :warning: *Although ready to be installed, the Checkout Identification app is currently an **open-beta project**. Bear in mind that due to that status, you can expect an accelerated state of evolution.*
```

<!-- DOCS-IGNORE:start -->
<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
[![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
<!-- ALL-CONTRIBUTORS-BADGE:END -->
<!-- DOCS-IGNORE:end -->

Under the app's name, you should explain the topic, giving a **brief description** of its **functionality** in a store when installed.

Next, **add media** (either an image of a GIF) with the rendered components, so that users can better understand how the app works in practice. 

> ![Media Placeholder](https://user-images.githubusercontent.com/52087100/71204177-42ca4f80-227e-11ea-89e6-e92e65370c69.png)

Reviewer: This is an instruction to clarify which info should be added to the doc - It should be replaced with real data!

### `block-2` props

| Prop name    | Type            | Description    | Default value                                                                                                                               |
| ------------ | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ---------- | 
| `XXXXX`      | `XXXXXX`       | XXXXXXXX         | `XXXXXX`        |

Prop types are: 

- `string` 
- `enum` 
- `number` 
- `boolean` 
- `object` 
- `array` 

When documenting a prop whose type is `object` or `array` another prop table will be needed. You can create it following the example below:

- `propName` object:

| Prop name    | Type            | Description    | Default value                                                                                                                               |
| ------------ | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ---------- | 
| `XXXXX`      | `XXXXXX`       | XXXXXXXX         | `XXXXXX`        |

> Remember to also use this Configuration section to  **showcase any necessary disclaimer** related to the app and its blocks, such as the different behavior it may display during its configuration. 

Reviewer: This is an instruction to clarify which info should be added to the doc - It should be replaced with real data!

## Modus Operandi *(not mandatory)*

There are scenarios in which an app can behave differently in a store, according to how it was added to the catalog, for example. It's crucial to go through these **behavioral changes** in this section, allowing users to fully understand the **practical application** of the app in their store.

> If you feel compelled to give further details about the app, such as it's **relationship with the VTEX admin**, don't hesitate to use this section. 

Reviewer: This is just an instruction to clarify which info should be added to the doc! If no info will be actually added regarding the app behavior, the Modus Operandi section can be removed!

## Customization

The first thing that should be present in this section is the sentence below, showing users the recipe pertaining to CSS customization in apps:

`In order to apply CSS customizations in this and other blocks, follow the instructions given in the recipe on [Using CSS Handles for store customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization).`

Thereafter, you should add a single column table with the available CSS handles for the app, like the one below. Note that the Handles must be ordered alphabetically.

| CSS Handles |
| ----------- | 
| `XXXXX` | 
| `XXXXX` | 
| `XXXXX` | 
| `XXXXX` | 
| `XXXXX` |

> If there are none, add the following sentence instead:

Reviewer: This is an instruction to add customization info to the doc! If no Handles are currently available, this part should be removed - only keep the No CSS Handles are available yet for the app customization.

> `No CSS Handles are available yet for the app customization.`

Reviewer: ```suggestion
No CSS Handles are available yet for the app customization.
```

<!-- DOCS-IGNORE:start -->

## Contributors ✨

Thanks goes to these wonderful people:

<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
<!-- prettier-ignore-start -->
<!-- markdownlint-disable -->
<!-- markdownlint-enable -->
<!-- prettier-ignore-end -->
<!-- ALL-CONTRIBUTORS-LIST:END -->

This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind are welcome!

<!-- DOCS-IGNORE:end -->

> 
> ---- 
> 
> Check out some documentation models that are already live: 
> - [Breadcrumb](https://github.com/vtex-apps/breadcrumb)
> - [Image](https://vtex.io/docs/components/general/vtex.store-components/image)
> - [Condition Layout](https://vtex.io/docs/components/all/vtex.condition-layout@1.1.6/)
> - [Add To Cart Button](https://vtex.io/docs/components/content-blocks/vtex.add-to-cart-button@0.9.0/)
> - [Store Form](https://vtex.io/docs/components/all/vtex.store-form@0.3.4/)

Reviewer: ```suggestion
```
